index.rst

```diff
@@ -11,6 +11,8 @@ The `Example product documentation`_ shows how to set up a Diátaxis structure i
 .. include_start
```

Comment:
Don't we want to get rid of these include directives since they're not being called anymore?